doc/examples/realtime_servo/src/pose_tracking_tutorial.cpp

```diff
@@ -0,0 +1,228 @@
+
+
```

Comment:
Unnecessary blank lines?

Instead, consider replacing them with a comment explaining what this demo does.

Reply:
Also, there are no comments explaining anything for the first 158 lines of this code!

Please add at least top level `@brief` docstring to all the class/struct definitions and functions created here. It will help users understand what this is all for.

Reply:
LOL, sorry about that , totally forgot to add comments in the code after I cleaned it up.